cart.html

```diff
@@ -0,0 +1,34 @@
+<!DOCTYPE html>
+<html lang="en"><head>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Document</title>
+    <link rel="stylesheet" href="styles/csrt.css">
+</head>
+<body>
+  <nav>
+    <div class="cartPageNavLogo">
+        <img src="assets/84426.svg">
+    </div>
+</nav>
+
+  <div class="cartPage">
```

Comment:
You need to follow a naming convention with classes' names.

See [these conventions](https://www.freecodecamp.org/news/css-naming-conventions-that-will-save-you-hours-of-debugging-35cea737d849/) and try to implement them, it will make your life easier XD!